src/FloatingActionButton/FloatingActionButton.spec.js

```diff
@@ -0,0 +1,31 @@
+/* eslint-env mocha */
+import React from 'react';
+import {shallow} from 'enzyme';
+import {assert} from 'chai';
+
+import FloatingActionButton from './FloatingActionButton';
+import getMuiTheme from '../styles/getMuiTheme';
+import ContentAdd from '../svg-icons/content/add';
+
+describe('<FloatingActionButton />', () => {
+  const muiTheme = getMuiTheme();
+  const shallowWithContext = (node) => shallow(node, {context: {muiTheme}});
+
+  describe('hover state', () => {
+    it('should reset the hover state when disabled', () => {
+      const wrapper = shallowWithContext(
+        <FloatingActionButton>
+          <ContentAdd />
+        </FloatingActionButton>
+      );
+      wrapper.setState({
```

Comment:
> because FloatingActionButton uses refs.

I'm not sure to understand this statement. Could you give more detail?
At least the following assert doesn't seem useful:
```js
assert.strictEqual(wrapper.state().hovered, true, 'should be hovered');
```